tests/manage/pv_services/test_pvc_disruptive.py

```diff
@@ -281,6 +291,11 @@ def test_pvc_disruptive(
         if interface == constants.CEPHFILESYSTEM:
             access_modes.append(constants.ACCESS_MODE_RWX)
 
+        # Modify access_modes list to create rbd `block` type volume with
+        # RWX access mode. RWX is not supported in non-block type rbd
+        if interface == constants.CEPHBLOCKPOOL:
+            access_modes.append(f'{constants.ACCESS_MODE_RWX}-Block')
```

Comment:
If we append with -fs, then https://github.com/red-hat-storage/ocs-ci/pull/901/files#diff-0a279209fc041923d854df2fc7b103e1R233  still considers as Block

Reply:
It is particular to this test case. -FileSystem won't be specified in this test. The conditional statements which check for Block is modified in recent commit.